### PR TITLE
data: Poonam1607

### PR DESCRIPTION
**Pull request by Poonam1607**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5753"></a>

### CODE REVIEW

**Review by swagataroy30 — COMMENTED** [review]
You don't need to add avatar, it will auto-pick your GitHub profile picture and also `"displayStatsPublic": true` is not required. So, please remove that.
 You can refer to the docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

File: data/Poonam1607.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Poonam Pawar",
+  "displayStatsPublic": true,
```
[the remark this answers is not in the document]
Comment (Poonam1607) [reply data/Poonam1607.json:3]: Do I remove this line?

**Comment by Poonam1607** [comment]
ThankYou @swagataroy30 I have committed your suggestions.

**Review by Ananya2001-an — APPROVED** [review]
Looks good 👍🏼

**Review by yoyoekta — APPROVED** [review]
Looks good to merge!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks awesome. Great collaboration everyone. 👍

**Pull request merged**